examples/job/create_and_submit_job.py

```diff
@@ -11,25 +11,46 @@
 # 
 # ## Import packages
 
-# In[2]:
-
-
-import json
-
+# In[1]:
+
+
+from IPython.display import JSON
+import os
+import sys
+
+# Install Pandas if it isn't present
+try:
+    import pandas as pd
+except ModuleNotFoundError:
+    import subprocess, sys
+    subprocess.call([sys.executable,'-m','pip','install','pandas==1.1.4'])
+    import pandas as pd
+# Install Tabulate if it isn't present
+try:
+    import tabulate
+except ModuleNotFoundError:
+    import subprocess, sys
+    subprocess.call([sys.executable,'-m','pip','install','tabulate==0.8.2'])
+    import tabulate
+# Install the API Client if it isn't present
+try:
+    import exabyte_api_client
+except ModuleNotFoundError:
+    import subprocess, sys
+    subprocess.call([sys.executable,'-m','pip','install','exabyte_api_client==2020.10.19'])
```

Comment:
Code copy-paste between notebooks seems less than ideal. Let's define a function (in utils?) import and reuse it for all notebooks adding max 2-3 lines to each?